ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -488,6 +496,35 @@ public static long toSecondsPrecisionInMs(long ms) {
   }
 
   /**
+   * Gets the {@link GetOptions} required by the request.
+   * @param restRequest the representation of the request.
+   * @return the required {@link GetOptions}. Defaults to {@link GetOptions#None}.
+   * @throws RestServiceException if the {@link RestUtils.Headers#GET_OPTIONS} is present but not recognized.
+   */
+  public static GetOptions getGetOptions(RestRequest restRequest)
+      throws RestServiceException {
+    GetOptions options = GetOptions.None;
```

Comment:
Always disliked how in our codebase we define enums in plural form. I understand that this is not a newly added enum, just saying that this should really have been `GetOption` and not `GetOptions` - as an instance only ever stores one option. Plural form would have made sense if we used "flags" and bit fields.